srml/council/src/seats.rs

```diff
@@ -463,17 +581,75 @@ impl<T: Trait> Module<T> {
 		// Prevent a vote from voters that provide a list of votes that exceeds the candidates length
 		// since otherwise an attacker may be able to submit a very long list of `votes` that far exceeds
 		// the amount of candidates and waste more computation than a reasonable voting bond would cover.
-		ensure!(candidates.len() >= votes.len(), "amount of candidate approval votes cannot exceed amount of candidates");
+		ensure!(candidates.len() >= votes.len(), "amount of candidate votes cannot exceed amount of candidates");
+
+		// Amount to be locked up.
+		// TODO: maybe this should be total - fee?
```

Comment:
Currently: you have 20. You pay 2 for a bad vote index. Your vote weight is still 20. Your lock is also 20.
Alternative: you have 20. You pay 2 for a bad vote index. Your vote weight is 18. Same for your lock.

I generally recommend triple checking this function before merging.